lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -219,6 +262,16 @@ export default function ContentSelector({
               Select PDF from Google Drive
             </LabeledButton>
           )}
+          {oneDriveFilesEnabled && (
+            <LabeledButton
+              onClick={showOneDrivePicker}
+              variant="primary"
+              data-testid="one-drive-button"
+            >
+              Select PDF from One Drive
```

Comment:
FYI it's OneDrive not One Drive. We might want to actually say Microsoft OneDrive as well